cmd/webhook/main.go

```diff
@@ -16,19 +16,27 @@ package main
 
 import (
 	"crypto/tls"
+	"context"
 	"flag"
 	"fmt"
 	"net/http"
+	"os"
 	"time"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
 	"github.com/fsnotify/fsnotify"
 	"github.com/golang/glog"
 	"github.com/k8snetworkplumbingwg/network-resources-injector/pkg/webhook"
 )
 
-const defaultClientCa = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+const (
+	defaultClientCa              = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+	customizedInjectionConfigMap = "nri-user-defined-injections"
```

Comment:
it would be nice if we didnt have the word "customized" in code since we are referring to this as "user-defined"

e.g `customizedInjectionConfigMap` -> `userDefinedInjectionConfigMap` and so on.

not something i would block on, your call if you want to do the extra refactor :)

Reply:
Done, it's easy to update the name than the code logic :-)